test/e2e/common/setup-kubestellar.sh

```diff
@@ -127,12 +127,12 @@ else
 popd
 
 : Waiting for OCM hub to be ready...
-kubectl wait controlplane.tenancy.kflex.kubestellar.org/its1 --for 'jsonpath={.status.postCreateHooks.its-hub-init}=true' --timeout 400s
-kubectl wait -n its1-system job.batch/its-hub-init --for condition=Complete --timeout 400s
-kubectl wait controlplane.tenancy.kflex.kubestellar.org/its1 --for 'jsonpath={.status.postCreateHooks.install-status-addon}=true' --timeout 400s
-kubectl wait -n its1-system job.batch/install-status-addon --for condition=Complete --timeout 400s
+kubectl --context "$HOSTING_CONTEXT" wait controlplane.tenancy.kflex.kubestellar.org/its1 --for 'jsonpath={.status.postCreateHooks.its-hub-init}=true' --timeout 400s
+wait-for-job-complete "$HOSTING_CONTEXT" its1-system its-hub-init 400
```

Comment:
That new command is a lot of complexity for little gain. What it adds should be added to the post-test debug dumping instead.